src/modules/ekf2/ekf2_params.c

```diff
@@ -484,7 +484,7 @@ PARAM_DEFINE_FLOAT(EKF2_MAG_GATE, 3.0f);
  * @bit 2 use declination as an observation
  * @reboot_required true
  */
-PARAM_DEFINE_INT32(EKF2_DECL_TYPE, 7);
+PARAM_DEFINE_INT32(EKF2_DECL_TYPE, 3);
```

Comment:
Should we even keep "bit 2 use declination as an observation"?

Reply:
I don't think that this option is needed, no.

Reply:
removed it